website/versioned_docs/version-26.x/TimerMocks.md

```diff
@@ -21,10 +19,8 @@ module.exports = timerGame;
 ```
 
 ```javascript
-// __tests__/timerGame-test.js
-'use strict';
-
-jest.useFakeTimers();
+// timerGame.test.js
```

Comment:
don't change the name of the file (not that it really matters, but it's extra noise in the diff)